data/json/itemgroups/Clothing_Gear/clothing.json

```diff
@@ -70,6 +70,18 @@
             ],
             "prob": 50
           },
+          {
+            "collection": [
+              {
+                "distribution": [
+                  { "item": "ar_glasses_t0", "prob": 80, "charges-max": 50 },
+                  { "item": "ar_glasses_t1", "prob": 20, "charges-max": 50 }
+                ],
+                "prob": 10
+              }
+            ],
+            "prob": 10
+          },
```

Comment:
You want to create a collection of either type of glasses and... what else? Unless I'm missing something, `collection` isn't needed here.

Reply:
I'm a bit confused by distributions VS collections. I look for there is a small change or spawning AR glasses, and within this chance, 80% of the time it would be the cheapest.

Should I just delete the collection and keep the distribution as it is now ?

Reply:
`Collection` means that game will try to spawn _every_ item in the item group, depending on their chance. `Distribution` means that game will try to spawn _only one_ item from the item group, depending on their chance.

> Should I just delete the collection and keep the distribution as it is now ?

Yes.

Reply:
Thanks a lot !